wideq/core.py

```diff
@@ -12,17 +13,38 @@
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
-GATEWAY_URL = 'https://kic.lgthinq.com:46030/api/common/gatewayUriList'
-APP_KEY = 'wideq'
-SECURITY_KEY = 'nuts_securitykey'
-DATA_ROOT = 'lgedmRoot'
-SVC_CODE = 'SVC202'
-CLIENT_ID = 'LGAO221A02'
-OAUTH_SECRET_KEY = 'c053c2a6ddeb7ad97cb0eed0dcb31cf8'
-OAUTH_CLIENT_KEY = 'LGAO221A02'
-DATE_FORMAT = '%a, %d %b %Y %H:%M:%S +0000'
-DEFAULT_COUNTRY = 'US'
-DEFAULT_LANGUAGE = 'en-US'
+GATEWAY_URL = (
+    "https://route.lgthinq.com:46030/v1/service/application/gateway-uri"
+)
+SECURITY_KEY = "nuts_securitykey"
+APP_KEY = "wideq"
+DATA_ROOT = "result"
+POST_DATA_ROOT = "lgedmRoot"
+RETURN_CODE_ROOT = "resultCode"
+RETURN_MESSAGE_ROOT = "returnMsg"
+SVC_CODE = "SVC202"
+OAUTH_SECRET_KEY = "c053c2a6ddeb7ad97cb0eed0dcb31cf8"
+OAUTH_CLIENT_KEY = "LGAO221A02"
+DATE_FORMAT = "%a, %d %b %Y %H:%M:%S +0000"
+DEFAULT_COUNTRY = "US"
+DEFAULT_LANGUAGE = "en-US"
+
+# v2
+API_KEY = "VGhpblEyLjAgU0VSVklDRQ=="
```

Comment:
To confirm, is the goal here to simultaneously support v1 and v2 of the API? If not, we can just delete all the stuff that is v1-specific.

Reply:
Hi - I'm lurking a bit here, however simultaneous v1/v2 support would help me. I have a mix of both in my home. I can do testing :) Cheers!

Reply:
Well, that is one of the things I was unsure about. Are there devices that ONLY support the V1 api? If so, then it would make sense to support both APIs at the same time. Otherwise, it'd make sense to ONLY support v2.